ui/src/config/section/compute.js

```diff
@@ -425,15 +425,18 @@ export default {
       icon: kubernetes,
       docHelp: 'plugins/cloudstack-kubernetes-service.html',
       permission: ['listKubernetesClusters'],
-      columns: () => {
+      columns: (store) => {
```

Comment:
nit - there is a global import on store, if we move to this format - should we remove that or refactor all usage per this new pattern?